tests/utils.py

```diff
@@ -13,8 +13,11 @@
 import transformers
 
 import QEfficient
+import QEfficient.transformers
+import QEfficient.transformers.modeling_utils

```

Comment:
`from QEfficient.transformers.modeling_utils import transform_lm` and use `transform_lm` in `L104 `directly.

Reply:
Thanks. Updated. Auto-import did this ;)